package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "eleventy-netlify-boilerplate",
```

Comment:
Let's update these fields to match the old site. We can also update the version number to 2 :)

Reply:
Great, updated.